docs/install/create-project-cookiecutter-volto.md

```diff
@@ -0,0 +1,183 @@
+---
+myst:
+  html_meta:
+    "description": "Create a project with cookiecutter-volto (experimental)"
+    "property=og:description": "Create a project with cookiecutter-volto (experimental)"
+    "property=og:title": "Create a project with cookiecutter-volto (experimental)"
+    "keywords": "Plone, Plone 6, create, project, install, cookiecutter-volto, pnpm"
+---
+
+(create-a-project-with-cookiecutter-volto-experimental-label)=
+
+# Create a project with cookiecutter-volto (experimental)
+
+This chapter describes the usage of the currently experimental {term}`cookiecutter` project [plone/cookiecutter-volto](https://github.com/plone/cookiecutter-volto/).
+The development of this add-on is done in isolation using a new approach using {term}`pnpm` workspaces and latest {term}`mrs-developer` and other Volto core improvements.
+In order to get started you will need {term}`pipx`, {term}`Make`, {term}`Node.js` and {term}`Docker` see {ref}`create-a-project-with-cookiecutter-volto-experimental-installation-label` section below.
+
+After that you can run following command to create your project:
+```shell
+pipx run cookiecutter gh:plone/cookiecutter-volto
+```
+
+(create-a-project-with-cookiecutter-volto-experimental-installation-label)=
+
+## Installation
+
+(create-a-project-with-cookiecutter-volto-experimental-installation-pipx-label)=
+
+### pipx
+
+Install {term}`pipx`.
+
+```shell
+pip install pipx
+```
+
+(create-a-project-with-cookiecutter-volto-experimental-installation-gnu-make-label)=
+
+### GNU make
+
+```{include} ../volto/contributing/install-make.md
+```
+
+(create-a-project-with-cookiecutter-volto-experimental-installation-node.js-label)=
+
+### Node.js
+
+First you'll need nvm, {term}`Node Version Manager`.
+
+#### nvm
+
+The following terminal session commands use `bash` for the shell.
+Adapt them for your flavor of shell.
+
+```{seealso}
+See the [`nvm` install and update script documentation](https://github.com/nvm-sh/nvm#install--update-script).
+For the `fish` shell, see [`nvm.fish`](https://github.com/jorgebucaran/nvm.fish).
+```
+
+1.  Create your shell profile, if it does not exist.
+
+    ```shell
+    touch ~/.bash_profile
+    ```
+
+2.  Download and run the `nvm` install and update script, and pipe it into `bash`.
+
+    ```shell
+    curl -o- https://raw.githubusercontent.com/creationix/nvm/v{NVM_VERSION}/install.sh | bash
+    ```
+
+3.  Source your profile.
+    Alternatively close the session and open a new one.
+
+    ```shell
+    source ~/.bash_profile
+    ```
+
+4.  Verify that the `nvm` version is that which you just installed or updated:
+
+    ```shell
+    nvm --version
+    ```
```

Comment:
Let's convert this entire section to an include file for easier maintenance, and not do copy paste. The include should go into a separate PR in https://github.com/plone/volto, and updated in both this PR and the volto PR in the following places:

- https://6.docs.plone.org/install/create-project.html#nvm
- https://6.docs.plone.org/volto/contributing/developing-core.html#nvm

I should have done that when I did the other includes, but this one may have been under revision.

Reply:
@stevepiercy added a pull request for plone/volto [here](https://github.com/plone/volto/pull/5962).

Reply:
@stevepiercy I am now using the nvm installation include file in create-project.html#nvm
create-project-cookiecutter-volto.html#nvm...
I just need to figure out how to update the volto submodule of this fork... and I will add further examples to create-project-cookiecutter-volto.html...

Reply:
@stevepiercy  the nvm installation include file is now used in create-project.html#nvm and
create-project-cookiecutter-volto.html#nvm...

Reply:
@animus888 sorry for the delay. We're awaiting revisions to the new cookiecutter that wraps other cookiecutters before pushing out new documentation of any cookiecutter. See https://github.com/plone/cookieplone. @sneridagh, @ericof, and @davisagli started work on that at the Axolote Sprint, and made significant progress, but I'm not sure where it stands.